src/components/Grid.js

```diff
@@ -0,0 +1,40 @@
+import Table from "react-bootstrap/Table";
+
+function Grid() {
```

Comment:
Here you use the `function` keyword while elsewhere you use the `const` keyword for functional component definition. could you standardize to use `const` all the time?